Jenkinsfile-bc

```diff
@@ -62,36 +53,43 @@ pipeline {
                 }
             }
         stage('Publish'){
-            //when { expression { params.skip_stage != true } }
             steps{
                 sh 'docker login -u bdcc10 -p ${DOCKER_PASS}'
                 sh 'docker tag graphql_service bdcc10/bdcc-at20_graphql_service:${GIT_COMMIT_HASH}'
                 sh 'docker push bdcc10/bdcc-at20_graphql_service:${GIT_COMMIT_HASH}'
 
                 }
             }
-
         stage ('DeployToDev') {
             steps {
                 sh 'echo ${GIT_COMMIT_HASH}'
                 sh 'export TAG_VERSION=${GIT_COMMIT_HASH}'
                 sh 'echo ${TAG_VERSION}'
                 sh 'docker-compose -f docker-compose.dev.bdcc.yaml up -d'
+                }
             }
-        }
          stage ('DeployToAuto') {
-            when { expression { params.skip_stage != true } }
             steps {
                 sh 'DOCKER_HOST=ssh://${TARGET_HOST} docker-compose -f docker-compose.dev.bdcc.yaml up -d'
             }
         }
     }
-
     post {
         always {
             sh 'docker image prune -a -f'
             sh 'docker system prune -a -f'
         }
+        success {
```

Comment:
Use environment variables to improve your email notification. always {}
Pipeline for ${BUILD_NUMBER} - has been executed - Status: ${BUILD_STATUS}
The Convert service build was executed ${BUILD_STATUS}
This is the log link

Reply:
Changes added to email notification

![image](https://github.com/AT20-DevOps/AT_20_GRAPHQL_SERVICE/assets/123556616/2893fd0a-4b02-400c-858b-f9aa9d260933)